includes/base/controls-stack.php

```diff
@@ -730,14 +732,15 @@ final public function get_scheme_controls() {
 	 * @return array Style controls.
 	 */
 	final public function get_style_controls( array $controls = null, array $settings = null ) {
-		// _deprecated_function( __METHOD__, '3.0.0' );
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.0.0' );
+
 
 		$controls = $this->get_active_controls( $controls, $settings );
 
 		$style_controls = [];
 
 		foreach ( $controls as $control_name => $control ) {
-			$control_obj = Plugin::$instance->controls_manager->get_control( $control['type'] );
+			$control_obj = Plugin::$instance->controls_manager->get_control( $control[ 'type' ] );
```

Comment:
Why?

Reply:
It is probably the intellj php code beautifier

Reply:
Please remove it, here and other files.

Reply:
remove the spaces within the square brackets?